runtime/testy/runtime.py

```diff
@@ -240,16 +242,27 @@ def startHibike(badThingsQueue, stateQueue, pipe):
   # badThingsQueue - queue to runtime
   # stateQueue - queue to stateManager
   # pipe - pipe from statemanager
+  def addPaths():
+    """Modify sys.path so we can find hibike.
```

Comment:
I'm personally not a fan of path-munging in python (as opposed to the env setup)... but I'll let it slide for now